code/modules/power/singularity/singularity.dm

```diff
@@ -399,32 +395,30 @@
 						  "<span class='userdanger'>You feel an inner fire as your skin bursts into flames!</span>")
 		C.adjust_fire_stacks(5)
 		C.IgniteMob()
-	return
 
 
 /obj/singularity/proc/mezzer()
 	for(var/mob/living/carbon/M in oviewers(8, src))
-		if(istype(M, /mob/living/brain)) //Ignore brains
+		if(M.stat != CONSCIOUS)
 			continue
-
-		if(M.stat == CONSCIOUS)
-			if (ishuman(M))
-				var/mob/living/carbon/human/H = M
-				if(istype(H.glasses, /obj/item/clothing/glasses/meson))
-					var/obj/item/clothing/glasses/meson/MS = H.glasses
-					if(MS.vision_flags == SEE_TURFS)
-						H << "<span class='notice'>You look directly into the [src.name], good thing you had your protective eyewear on!</span>"
-						return
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(istype(H.glasses, /obj/item/clothing/glasses/meson))
+				var/obj/item/clothing/glasses/meson/MS = H.glasses
+				if(MS.vision_flags == SEE_TURFS)
+					H << "<span class='notice'>You look directly into \the \
```

Comment:
i'm going to throw you and your linebreaks into the goddamn star you're trying to pr